server/config/connectDB.ts

```diff
@@ -0,0 +1,15 @@
+import mongoose from "mongoose";
+import envProvider from "../utils/envProvider.utils";
```

Comment:
moved this file to config folder and change name `envProvider` to `appConfig`

Reply:
done